### PR TITLE
fix: remove mariner 1.0 from vhd-content-test suite

### PR DESCRIPTION
**Pull request by SeanDougherty**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
Removing VHD testing in 1.0 is not a regression in behavior since we weren't testing 1.0 before
this week. Going forward we're focusing on testing 2.0, and don't want to add workarounds within
the tests to make them support 1.0

**Which issue(s) this PR fixes**:

N/A

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:
1.0 not passing was uncovered after adding #3058

**Release note**:
```
none
```

### CODE REVIEW

**Review by hbeberman — APPROVED** [review]
LGTM.

Not needed for this PR, but this script should be refactored slightly to at least stash the script arguments into named variables so that these function calls are readable.

**Comment by SeanDougherty** [comment]
Electing to remove 1.0 testing. Going forward we're focusing on testing 2.0, and don't want to add workarounds within
the tests to make them support 1.0

**Review by hbeberman — APPROVED** [review]

File: vhdbuilder/packer/test/run-test.sh [hunk 1]
```diff
@@ -11,8 +11,8 @@ TEST_VM_ADMIN_PASSWORD="TestVM@$(date +%s)"
 set -x
 
 if [ "$OS_TYPE" == "Linux" ]; then
-  if [ "$IMG_SKU" == "20_04-lts-cvm" ]; then
-    echo "Skipping tests for CVM 20.04"
+  if [ "$IMG_SKU" == "20_04-lts-cvm" ] || [ "$IMG_SKU" == "cbl-mariner-1" ]; then
```
Comment (alexeldeib) [inline vhdbuilder/packer/test/run-test.sh:14]: don't we have v1 gen2 now as well? or is that covered by this?

**Review by alexeldeib — APPROVED** [review]

File: vhdbuilder/packer/test/run-test.sh (hunk 1 above)
Comment (SeanDougherty) [reply vhdbuilder/packer/test/run-test.sh:14]: Good call out, I have sent up an update and will validate against the pipeline before merging

**Pull request merged**